src/main/java/net/twasi/obsremotejava/OBSRemoteController.java

```diff
@@ -378,6 +394,42 @@ public void saveReplayBuffer(Callback<SaveReplayBufferResponse> callback) {
         communicator.saveReplayBuffer(callback);
     }
 
+    public void playPauseMedia(String sourceName, Boolean playPause, Callback<PlayPauseMediaResponse> callback) {
```

Comment:
Getting an "invalid request type" message when using this, can you repro?

onMessage: {
    "error": "invalid request type",
    "message-id": "4",
    "status": "error"
}